playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -298,6 +298,7 @@ EDXAPP_FEATURES:
   ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES: "{{ EDXAPP_ENABLE_READING_FROM_MULTIPLE_HISTORY_TABLES }}"
   SHOW_HEADER_LANGUAGE_SELECTOR: "{{ EDXAPP_SHOW_HEADER_LANGUAGE_SELECTOR }}"
   SHOW_FOOTER_LANGUAGE_SELECTOR: "{{ EDXAPP_SHOW_FOOTER_LANGUAGE_SELECTOR }}"
+  ONLY_THIRD_PARTY_AUTH: "{{ EDXAPP_ONLY_THIRD_PARTY_AUTH }}"
```

Comment:
Have corresponding changes been made to cms/envs/aws.py in platform?

Reply:
yes, we already have that as part of CMS AAD work that racoongang has done.